ffi/framebuffer_SDL2_0.lua

```diff
@@ -33,6 +22,44 @@ function framebuffer:init()
     framebuffer.parent.init(self)
 end
 
+function framebuffer:resize(w, h)
+    local w = w or SDL.w
+    local h = h or SDL.h
+
+    if not self.dummy then
+        self:_newBB(w, h)
+    else
+        self.bb = BB.new(600, 800)
```

Comment:
In that else (if it ever happens), aren't the `self.bb` and `self.sdl_bb` (otherwise freed by _newBB()) not freed ?
Nothing else obvious for the cause of memory leaks...

Reply:
The else is just for unit tests, but true. I was thinking it should just be a no-op and then I changed my mind 'cause maybe you'd want to do a unit test on some resizing type stuff and then I apparently ended up doing neither in my actual commit. >_>

Reply:
Actually I said that as if I were sure, but I don't know if this self.dummy is even used anywhere.